src/test/java/net/snowflake/client/config/SFClientConfigParserTest.java

```diff
@@ -40,6 +42,22 @@ public void testloadSFClientConfigValidPath() {
     }
   }
 
+  @Test
+  public void testloadSFClientConfigValidPathWithUnknownProperties() {
+    Path configFilePath = Paths.get("config.json");
+    try {
+      Files.write(configFilePath, CONFIG_JSON_WITH_UNKNOWN_PROPS.getBytes());
+      SFClientConfig actualConfig =
+          SFClientConfigParser.loadSFClientConfig(configFilePath.toString());
+      assertEquals("info", actualConfig.getCommonProps().getLogLevel());
+      assertEquals("/jdbc.log", actualConfig.getCommonProps().getLogPath());
+
+      Files.delete(configFilePath);
```

Comment:
should we move it to `finally`?

Reply:
I have cleaned up the code a little bit.